includes/common.php

```diff
@@ -634,6 +634,25 @@ function format_bi($value, $round = '2', $sf = '3')
     return number_format(round($value, $round), $sf, '.', '').$ext;
 }
 
+function format_graph_bi($current, $total, $round = 2)
```

Comment:
So very close, you can't have this function accept 2 values to format. It should be a totally generic function that just accepts an int value and round value. You'll have to call the function twice but that's fine.